src/applications/personalization/dashboard/components/cerner-widgets.js

```diff
@@ -1,9 +1,38 @@
 import React from 'react';
-import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
-
+import PropTypes from 'prop-types';
 import { mhvUrl } from 'platform/site-wide/mhv/utilities';
 import { getCernerURL } from 'platform/utilities/cerner';
 
+// Helper component that takes an array of facility names and a separator string and returns some JSX to style the list of facility names.
+const FacilityList = ({ facilities, separator }) => {
+  const newArray = [];
+  // first make an array that alternates between a facility name and the
+  // separator word wrapped in a <span> for styling purposes
+  facilities.forEach(el => {
+    newArray.push(el);
+    newArray.push(
+      <span className="vads-u-font-weight--normal vads-u-font-size--base">
+        {separator}
+      </span>,
+    );
+  });
+  // we don't need the last separator in the array
+  newArray.pop();
+  // Then map over the array we just made, converting it to JSX
+  return (
+    <>
+      {newArray.map((el, i) => {
+        return <strong key={i}>{el}</strong>;
+      })}
+    </>
+  );
+};
+
+FacilityList.propTypes = {
+  facilities: PropTypes.string,
```

Comment:
Shouldn't this prop type for `facilities` be an array?

Reply:
lol yes. Let me fix